api/src/main/java/com/cloud/event/EventTypes.java

```diff
@@ -745,7 +745,7 @@ public class EventTypes {
     public static final String EVENT_FILESHARE_CREATE = "FILESHARE.CREATE";
     public static final String EVENT_FILESHARE_START = "FILESHARE.START";
     public static final String EVENT_FILESHARE_UPDATE = "FILESHARE.UPDATE";
-    public static final String EVENT_FILESHARE_CHANGE_SVC_OFFERING = "FILESHARE.CHANGE.SERVICE.OFFERING";
+    public static final String EVENT_FILESHARE_CHANGE_SVC_OFFERING = "FILESHARE.CHANGE.SVC.OFFERING";
```

Comment:
why make this name more obscure?

Reply:
The string length was getting more than that allowed in the db table. But have renamed it now to SHAREDFS.CHANGE.SERVICE.OFFERING